cachi2/core/package_managers/yarn/main.py

```diff
@@ -154,4 +154,11 @@ def _undo_changes(project: Project) -> None:
 
```

Comment:
**Commit message**:
It actually took me a while to figure out why we're setting the global cache options instead of just overriding the local cache for the build. Because of that I'd like to see some bits from the https://github.com/chmeliik/berryscary/commit/1275d17d761090fa6999000eb1991ad4e074eacc commit message here so that even after a few months we have this crucial information tracked centrally.